org.gnome.Photos.json

```diff
@@ -115,6 +114,74 @@
                 }
             ]
         },
+        {
+            "name": "gssdp",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                "type": "git",
+                "url": "https://gitlab.gnome.org/GNOME/gssdp.git",
+                "commit": "6eeaf27aec6ff11f325ff633b52bd56dbc1435aa",
+                "tag": "gssdp-1.6.0"
+            }
+            ]
+        },
+        {
+            "name": "gupnp",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                "type": "git",
+                "url": "https://gitlab.gnome.org/GNOME/gupnp.git",
+                "commit": "f05648e240c99be32911fb0c6de84e5954470e56",
+                "tag": "gupnp-1.6.0"
+            }
+            ]
+        },
+        {
+            "name": "gupnp-av",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                "type": "git",
+                "url": "https://gitlab.gnome.org/GNOME/gupnp-av.git",
+                "commit": "86f148ebb920a8113190fd3e6cae636c9fdd3ca7",
+                "tag": "gupnp-av-0.14.1"
+            }
+            ]
+        },
+        {
+            "name": "gupnp-dlna",
+            "config-opts": [
+                "--disable-Werror"
+            ],
+            "sources": [
+                {
+                "type": "git",
+                "url": "https://gitlab.gnome.org/GNOME/gupnp-dlna.git",
+                "commit": "cc09daae47ef0e030895edbeba58636c76f53b3d",
+                "tag": "gupnp-dlna-0.12.0"
+            }
+            ]
+        },
+        {
+            "name": "dleyna",
+            "buildsystem": "meson",
+            "config-opts": [
+                "-Dserver_name=com.intel.dleyna-renderer",
```

Comment:
This is the wrong server name, you won't be able to own that. See the totem or rhythmbox manifests.

Reply:
I made some incorrect assumptions. Fixed (I think)